docs/2.8.0/docs/canton/usermanual/manage_domain_entities.rst

```diff
@@ -178,3 +178,10 @@ dynamically onboarding new sequencers (supported by Fabric and Ethereum sequence
    :start-after: user-manual-entry-begin: DynamicallyOnboardSequencerWithSeparateConsoles
    :end-before: user-manual-entry-end: DynamicallyOnboardSequencerWithSeparateConsoles
    :dedent:
+
+A newly onboarded sequencer only serves events sequenced more recently than the "onboarding snapshot", or more precisely events with timestamps more recent than the
+timestamp at which the new sequencer's signing key becomes valid. If a participant (or mediator) connects to a newly onboarded sequencer too soon, the sequencer
```

Comment:
The new sequencer's signing key normally becomes valid the very moment the sequencer becomes onboarded. So you don't really need this second clause in the sentence. However, what's missing here is that the some events may belong to transactions that were initiated before the sequencer was onboarded. And those events cause then such tombstones. So how about mentioning this correlation of "events" to earlier transactions? Then the maximum decision time is also motivated.

Reply:
Thanks, Andreas